api-admin/src/main/java/bio/terra/pearl/api/admin/service/enrollee/EnrolleeImportExtService.java

```diff
@@ -72,6 +102,31 @@ public void delete(
       throw new PermissionDeniedException(
           "Import Id does not belong to the given study environment");
     }
-    importService.delete(id, CascadeProperty.EMPTY_SET);
+    importService.deleteEnrolleesByImportId(id);
+    importService.updateStatus(id, ImportStatus.DELETED);
+  }
+
+  public void deleteImportItem(
```

Comment:
As of now not deleting a single ImportItem from UI. Maybe can be used in future.